src/content/h3/engine/ai/readme.md

```diff
@@ -91,15 +99,20 @@ AI are spawned in as part of a squad, which is then composed of fireteams, each
 
 # Combat
 
-When an AI enters combat their reaction time and if they can fire their weapon at this range is determined value values in the [character](~) tag, should the AI perform a dodge, be stunned, hard pinged or engage in a movement hint of some type, their accuracy bounds and firing pattern will be reset. 
+When an AI enters combat their reaction time and if they can fire their weapon at this range is determined value values in the [character](~) tag, should the AI perform a dodge, be stunned, hard pinged or engage in a movement hint of some type, their accuracy bounds and firing pattern will be reset to their initial values. 
 
 Based on certain scenarios the AI can engage in many behaviors to aide them in battle and appearing lifelike, this can include action such as a drawn out wide-spanning search for a hidden target, lobbing a grenade at cover where an enemy is suspected, suppressing fire on cover where a target was just seen entering, shooting dead bodies, checking on dead allies and giving each other orders (such as throwing grenades or starting a search with the ordered AI). 
 
 An AI without a valid [objective and/or valid task](~objectives) will have a green triangle over their head.
 
 # Limitations
 
-AI cannot move while throwing grenades, but they can throw a grenade and then initiate a leap and finish throwing the grenade while airborne. AI cannot melee while moving, outside of the movement the animation itself provides. 
-AI cannot shoot or melee while jumping (jumppack brute leaps are a type of jump).
-Information such as location of enemies is not passed between squads, even if they are right next to each other, however that information will be passed along if the squads share a task together.
-AI under normal circumstances do not use the melee damage assigned in [weapon](~) tags, instead they use the melee [damage](~damage_effect) assigned in the AI's [biped](~)
\ No newline at end of file
+- AI cannot move while throwing grenades, but they can throw a grenade and then initiate a leap and finish throwing the grenade while airborne. 
```

Comment:
If this is meant to be a bulleted list then the empty lines should be removed between each point again. With the empty lines, this is interpreted as a series of 1-item lists. If you’re including the blank lines for readability and spacing then maybe a list is not the right format for this info, and a series of paragraphs would be better. I think the list is fine though.